README.md

```diff
@@ -63,6 +63,12 @@ A good starting point for troubleshooting is:
 
 If you're still having trouble, post an issue so we can look into it.
 
+## Tested with the following RN versions
+
+Folks using this module have confirmed the library works with the following React Native versions. If your version is not on the list and you have confirmed it works as expected please create a PR (you can even do it on github.com)
+
+- 0.42.0
```

Comment:
Confirmed working with 0.43.x and 0.44.x :+1:

I understand that as of React 0.46.x there is a breaking change to the bundler, but I'm not sure if react-native-schemes-manager will have any issues with this upgrade yet.

Full context: https://github.com/facebook/react-native/releases/tag/v0.46.4